src/vs/base/browser/ui/list/listView.ts

```diff
@@ -188,7 +188,7 @@ export class ListView<T> implements ISpliceable<T>, IDisposable {
 	private currentDragFeedbackDisposable: IDisposable = Disposable.None;
 	private onDragLeaveTimeout: IDisposable = Disposable.None;
 
-	private disposables: IDisposable[];
+	private disposables: DisposableStore;
```

Comment:
Please make this property `readonly`. This helps prevent the disposables registered on a `DisposableStore` from being lost with a re-assignment

Reply:
Thanks. Fixed.